### ISSUE TITLE
feature request: live search

### ISSUE THREAD

**Issue by landsman** [jackocnr/intl-tel-input#990]
I would very much appreciate it option to search: prefix / country name.
Did you think about it?

**Comment by jackocnr** [comment]
We actually already have this - if you open the dropdown and type a country name you will see it is selected. Does that solve your problem?

**Comment by landsman** [comment]
@jackocnr oh, it's working. I expected another input where can I write and list will be filtered.

**Comment by jackocnr** [comment]
Yeah it would be nice, but it would make the UI way more complicated and I'd rather keep it simple for now.

**Comment by landsman** [comment]
I understand but many people just do not know about this feature if they do not see input or some tooltip. What about tooltip or title?

**Comment by jackocnr** [comment]
Well we already have a title on the selected flag which you can see if you hover it for long enough.

As for a tooltip, or some other kind of explanation, I would leave this up to the website developer to add themselves, in a style that fits the design of their website. You can put a note above the input, or as you say, you could show a tooltip - we expose the `open:countrydropdown` event so you can listen for that.

**Comment by landsman** [comment]
This is example of input on https://business.revolut.com:

![revolut_1](https://user-images.githubusercontent.com/3184228/67204780-8fc0af80-f40e-11e9-99eb-60b7cc3774c5.png)

![revolut_2](https://user-images.githubusercontent.com/3184228/67204788-93eccd00-f40e-11e9-8887-c88368a7cd97.png)

![revolut_3](https://user-images.githubusercontent.com/3184228/67204793-964f2700-f40e-11e9-8779-9e0feadd105c.png)

Good solution is on personal: http://revolut.com too:

![revolut_5](https://user-images.githubusercontent.com/3184228/67204940-e62dee00-f40e-11e9-809b-50ee15eb6d78.png)

![revolut_6](https://user-images.githubusercontent.com/3184228/67204943-e7f7b180-f40e-11e9-830d-564771afb705.png)

**Comment by jackocnr** [comment]
See new option `countrySearch` in v18.5.1. Let me know what you think!
